README.md

```diff
@@ -196,6 +196,53 @@ for debugging, if the error is not about dependencies.
 *aws-nuke* retries deleting all resources until all specified ones are deleted
 or until there are only resources with errors left.
 
+### Specifying Resource Types to Delete
+
+*aws-nuke* deletes a lot of resources and there might be added more at any
+release. Eventually, every resources should get deleted. You might to restrict
+which resources to delete. There are multiple ways to configure this.
+
+One way are filters, which already got mentioned. This requires to know the
+identifier of each resource. It is also possible to prevent whole resource
+types (eg `S3Bucket`) from getting deleted with two methods.
+
+* The `--target` flag limits nuking to the specified resource types.
+* The `--exclude` flag prevent nuking of the specified resource types.
+
+It is also possible to configure the resource types in the config file like in
+this example:
+
+```
+---
+regions:
+  - "eu-west-1"
+account-blacklist:
+- 1234567890
+
+resource-types:
+  target:
+  - S3Object
+  - S3Bucket
+  - IAMRole
+
+accounts:
+  555133742:
+    resource-types:
+      exclude:
+      - S3Bucket
+```
+
+If targets are specified in multiple places (eg CLI and account specific), then
+a resource type must be specified in all places. In other words each
+configuration limits the previous ones.
+
+If an exclude is defined in any place, then it will be ignored in any case.
```

Comment:
Consider being more explicit, along the lines of:
> If an exclude is used, then all its resource types will not be deleted.

Reply:
fxied